examples/api/test/material/chip/deletable_chip_attributes.on_deleted.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/chip/deletable_chip_attributes.on_deleted.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('RawChip.deleteIconBoxConstraints updates delete icon size constraints', (WidgetTester tester) async {
```

Comment:
This test description is from the other DeletableChipAttributes examples. Can you update it?

Reply:
Arf sorry about that :(

I changed it in [test: Update description](https://github.com/flutter/flutter/pull/152361/commits/319c904c18646c625e971d652e6b8e00bce83b54)